packages/create-next-app/create-app.ts

```diff
@@ -64,8 +64,8 @@ export async function createApp({
     try {
       repoUrl = new URL(example)
     } catch (error: unknown) {
-      const err = error as Error & { code: string | undefined }
-      if (err.code !== 'ERR_INVALID_URL') {
+      const err = error as Error
+      if (!(err instanceof TypeError)) {
```

Comment:
can you add more context on why we filter out `TypeError`?

Reply:
added a comment

Reply:
done